doc/classes/Object.xml

```diff
@@ -441,7 +442,8 @@
 			<return type="bool" />
 			<argument index="0" name="class" type="String" />
 			<description>
-				Returns [code]true[/code] if the object inherits from the given [code]class[/code].
+				Returns [code]true[/code] if the object inherits from the given [code]class[/code]. See also [method get_class].
+				[b]Note:[/b] [method is_class] does not take [code]class_name[/code] declarations into account. If the object has a [code]class_name[/code] defined and [method is_class] is used to check for inheritance, [method is_class] will return [code]false[/code] unless the base class' name is used as a parameter instead.
```

Comment:
```suggestion
				[b]Note:[/b] [method is_class] does not take [code]class_name[/code] declarations into account. If the object has a [code]class_name[/code] defined, [method is_class] will return false for that name.
```

A bit less wordy and more accurate (since it will also return true for the base class of the base class, etc).